src/components/validationStatus/ValidationStatus.vue

```diff
@@ -86,7 +90,7 @@ export default {
   components: {
     FontAwesomeIcon,
   },
-  props: ['validationErrors', 'warnings'],
+  props: ['validationErrors', 'warnings', 'owner'],
```

Comment:
The `owner` prop doesn't appear to be used in this component; did you mean to pass it to modeler instead?

Reply:
removed